libs/cpp-ledger/filecoin/types/version_info.hpp

```diff
@@ -25,6 +25,34 @@ namespace ledger::filecoin {
       return !(*this == other);
     }
 
+    inline bool operator<(const VersionInfo &other) const {
+      if (major < other.major) {
+        return true;
+      }
+
+      if (minor < other.minor) {
+        return true;
+      }
+
+      if (patch < other.patch) {
+        return true;
+      }
+
+      return false;
```

Comment:
example `(2 0 0) < (1 1 0)` is wrong.  
see `core/common/cmp.hpp` for correct implementation